### PR TITLE
DbLedgerStorage -- Added config options in bk_server.conf

### PR DESCRIPTION
**Pull request by merlimat**
Added commented config options with default values.

### CODE REVIEW

File: bookkeeper-server/conf/bk_server.conf [hunk 1]
```diff
@@ -574,3 +574,42 @@ zkEnableSecurity=false
 # This provides the plugin run extra services along with a bookie server.
 #
 # extraServerComponents=
+
+
+#############################################################################
```
Comment (sijie) [inline bookkeeper-server/conf/bk_server.conf]: Can you add those settings to  documentation? https://github.com/apache/bookkeeper/blob/master/site/_data/config/bk_server.yaml

**Comment by eolivelli** [comment]
It seems that master is not compilable on travis, but it is not due to this patch

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
close/reopen to trigger travis ci

File: bookkeeper-server/conf/bk_server.conf (hunk 1 above)
Comment (merlimat) [reply bookkeeper-server/conf/bk_server.conf]: Sure, will do that before merging

**Comment by merlimat** [comment]
> +1 for me. I think that it will be very important to write at least one page of docs about this great feature 

Yes, will do that before 4.7 is ready

**Pull request closed without merging**